resources/puppetlabs/lein-ezbake/template/foss/ext/debian/ezbake.init.erb

```diff
@@ -62,6 +64,11 @@ do_start()
     <%= action %>
     <% end -%>
 
+    if [ ! -d "/var/run/puppetlabs/${REALNAME}" ] ; then
```

Comment:
no strong feelings. if the directory exists, hopefully we created it and chowned it.